<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatBox</class>
 <widget class="QDialog" name="ChatBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="input">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>621</width>
     <height>181</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter键发送</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="emoji">
      <property name="text">
       <string>表情</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="modify_color">
      <property name="text">
       <string>字体颜色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="modify_font">
      <property name="text">
       <string>调整字体</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="upload_file">
      <property name="text">
       <string>上传文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="view">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
